src/Data/Sum/Properties.agda

```diff
@@ -33,3 +34,19 @@ module _ {a b} {A : Set a} {B : Set b} where
 
   swap-involutive : swap {A = A} {B = B} ∘ swap ≗ id
   swap-involutive = [ (λ _ → refl) , (λ _ → refl) ]
+
+module _ where
+  private
+    variable
+      a b c d e : Level
+      A : Set a
+      B : Set b
+      C : Set c
+      D : Set d
+      E : Set e
+
+  [,]-map-comm : ∀ {f′ : C → E} {f : A → C}
```

Comment:
Looking at `Data.List.Properties` it appears we usually write out `commute` in full?

Reply:
Also sets should occur in alphabetical order `A`, `B`, `C` etc. and `f` should appear before `f'`. We also normally use \prime instead of `. (Not particularly in favour of that last one but best to be consistent!)

Reply:
I chose to use ′ as this was the apostrophe used in [_,_]′ which I'm proving my fact about, so it seemed appropriate to use that one but I can change it if you think that would be better.

Reply:
Hmm at one point we had a drive to eliminate any `'` from the library but apparently they've crept in again *sigh* I'd change it to prime for the moment. I've opened an issue for it #995 in the general case.

Reply:
I changed the apostrophes of my f's and g's to \prime.
I couldn't however have the sets appear in alphabetical order due to the nature of the function arguments but I don't believe this to be a problem. They are now as alphabetical as they can be.

Reply:
Also, I proved another very similar property along with this one called map-commute.
I don't think this is the proper name to call it as it's not commutativity but I don't really know what to call this binary operation property.
I believe there should be a new entry in Algebra.Definitions for the following Op2 interaction:
```
(w . x) * (y . z) = (w * y) . (x * z)
```

Reply:
> I believe there should be a new entry in Algebra.Definitions for the following Op2 interaction

Hmm that looks useful though I have no idea what to call it. Feel free to open another PR.